package/MDAnalysis/topology/objects.py

```diff
@@ -0,0 +1,959 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
```

Comment:
Is `topology/objects.py` identical to `core/topologyobjects.py`? Is one a duplicate??